ingress/controllers/nginx/nginx.tmpl

```diff
@@ -156,6 +156,12 @@ http {
     }
     {{ end }}
 
+    {{/* build all the required rate limit zones. Each annotation requires a dedicated zone */}}
+    {{/* 1MB -> 16 thousand 64-byte states or about 8 thousand 128-byte states */}}
+    {{ $zone := range (buildRateLimitZones .servers) }}
```

Comment:
@louis-paul this is fixed in https://github.com/kubernetes/contrib/pull/1104. 
(just in case this PR is not published yet)